src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,86 @@
+package duke;
+import java.util.ArrayList;
+import java.util.List;
+import duke.task.Task;
+
+/**
+ * Class to abstract the List of the Tasks
+ */
+public class TaskList {
+
+    private final List<Task> taskList;
```

Comment:
Maybe tasks is more appropriate than taskList since it is a collection?

Reply:
Sure